### PR TITLE
FIX => Wrong redesignate popup on item search page

### PR DESCRIPTION
**Pull request by abulsayyad123**
Hi Team,
This is fix for wrong popup for designated item on item search page. The issue was due to previous queryparams not getting cleaned up. Issue is fixed.
Please review the PR and let me know if any change is required.
Thanks.

### CODE REVIEW

File: app/routes/items/search_order.js [hunk 1]
```diff
@@ -84,5 +84,12 @@ export default AuthorizeRoute.extend({
     }
     controller.set("searchText", "");
     controller.set("backLinkPath", this.get("itemDesignateBackLinkPath"));
+  },
+
+  resetController(controller, isExiting) {
+    if (isExiting) {
+      controller.set("partial_qty", undefined);
```
Comment (Darshan-Chauhan) [inline app/routes/items/search_order.js]: why are we setting undefined?? I would very much prefer null

Comment (abulsayyad123) [reply app/routes/items/search_order.js]: On doing `null` url does not remove these params and remains with value `Nan`. It removed it on assigning `undefined`.

Comment (patrixr) [reply app/routes/items/search_order.js]: Ember object should really have an `unset` method. There's no clear way of deleting a key :/

Comment (patrixr) [reply app/routes/items/search_order.js]: Just noticed this `controller.set("isSet", undefined);` 

--> could we make the variable name of `isSet` a bit more explicit ? I'm not really sure that is set or not 😆

Comment (abhinav098) [reply app/routes/items/search_order.js]: right @patrixr, the `isSet` property is not very self-explanatory. @abulsayyad123 could you please make this something understandable ?

Comment (abhinav098) [reply app/routes/items/search_order.js]: @patrixr , just got to know
set is collection of packages.. eg =>  Dining table is set which is made of 4 chairs and 1 table
you will find set_item_id column on package table maybe. 
isSet checks that the package is a set or not. 

cc - @namrataukirde @abulsayyad123

**Pull request merged**